versions/development/SPEC.md

```diff
@@ -358,6 +359,65 @@ Strings can also contain the following types of escape sequences:
 * A hexadecimal escape code starts with `\x`, followed by 2 hexadecimal digits `0-9a-fA-F`. 
 * A unicode code point starts with `\u` followed by 4 hexadecimal characters or `\U` followed by 8 hexadecimal characters `0-9a-fA-F`.
 
+##### Multi-line Strings
+
+Strings that begin and end with three consecutive single- or double-quotes may span multiple lines. The opening quotes of a multi-line string may be followed by whitespace (optional) and must have a newline before the first non-whitespace character; these leading whitespace/newline characters are removed. All subsequent non-empty lines are then used to determine the multi-line string's *common leading whitespace* - the minimum number of whitespace characters occuring before the first non-whitespace character in a line or the end of the line, whichever comes first. This common leading whitespace is stripped from the beginning of all the lines in the multi-line string.
+
+```wdl
+# These strings are equivalent. The middle lines strings B, C, and D are empty and so do not count
+# towards the common leading whitespace determination.
+
+String multi_line_A = """
+    this is a
+  a multi-line string"""
```

Comment:
"a" shouldn't appear both at the end of one line and the beginning of the next here, right?

Reply:
Doesn't multi_line_A start with two spaces after removal of common leading whitespace? (which isn't the same as B/C/D)

Reply:
good catch - fixed